README.md

```diff
@@ -74,6 +74,10 @@ If you know you will not be using the `$container` parameter or the `$getPreviou
     }
 ```
 
+A factory method returns the instance of the service it represents. A factory method does not need to ensure the unicity of the service returned. The container is in charge of calling the factory only once and then returning always the same service.
```

Comment:
Your proposal is indeed much better. Let me try to improve a bit:

> Each factory method is responsible for returning a given entry of the container. Nothing should be cached by service providers, this is the responsibility of the container.

Good enough?